akka-http/src/main/scala/akka/http/scaladsl/common/StrictForm.scala

```diff
@@ -40,6 +40,9 @@ sealed abstract class StrictForm {
 }
 
 object StrictForm {
+  // TODO: make timeout configurable
+  val toStrictTimeout = 10.seconds
```

Comment:
Not so good :-1: ;)

Reply:
(not sure if there's an easy better solution, though)

Reply:
Since we're dealing with directives it seems like it should be possible to read it from the config... something for another PR though?